packages/core/src/components/graphs/area.ts

```diff
@@ -41,7 +41,17 @@ export class Area extends Component {
 		const { cartesianScales } = this.services;
 
 		const orientation = cartesianScales.getOrientation();
-		const areaGenerator = area().curve(this.services.curves.getD3Curve());
+		const areaGenerator = area().curve(this.services.curves.getD3Curve())
+		.defined((datum: any, i) => {
+			if (boundsEnabled) {
+				const rangeIdentifier = cartesianScales.getRangeIdentifier();
+				const value = datum[rangeIdentifier];
+				if (value === null || value === undefined) {
+					return false;
+				}
+			}
+			return true;
+		});
```

Comment:
seems like the use of `defined()` is missing in the area chart, can we remove the `if (boundsEnabled)` check to make sure that this also works for regular area charts? Thanks

Reply:
Yep sure should be done now! 
<img width="742" alt="Screenshot 2021-03-30 at 16 43 56" src="https://user-images.githubusercontent.com/44590418/113017754-b80e1b00-9177-11eb-82b5-a83f830311db.png">